examples/websocket-relay/relay-app/src/components/WorkflowDetail.svelte

```diff
@@ -14,13 +15,13 @@
     <div class="p-4 overflow-y-auto scrollbar-hide">
       <div class="text-lg uppercase font-medium">Workflow One</div>
       <div class="jsonview">
-        <JsonView json={workflowOneJson} />
+        <!-- <JsonView json={workflowOneJson} /> -->
```

Comment:
I probably commented these `JsonView`s out. We should bring them back. They should display the JSON for the workflows when a user clicks the stack icon at the far right in the header.

Reply:
done! 
<img width="1793" alt="image" src="https://github.com/ipvm-wg/homestar/assets/1179291/6345d2e1-70e7-4275-a10e-a12ddfad194c">